python/servo/build_commands.py

```diff
@@ -324,6 +325,21 @@ def build(self, target=None, release=False, dev=False, jobs=None,
                     for ssl_lib in ["libcryptoMD.dll", "libsslMD.dll"]:
                         shutil.copy(path.join(env['OPENSSL_LIB_DIR'], "../bin" + msvc_x64, ssl_lib),
                                     servo_exe_dir)
+                    if os.environ["VisualStudioVersion"] == "14.0":
+                        vs_dir = os.environ["VS140COMNTOOLS"]
+                        vs_dll_dir = path.join(vs_dir, "..", "IDE", "Remote Debugger", os.environ["PLATFORM"].lower())
+                        for vs_dll in ["api-ms-win-crt-runtime-l1-1-0.dll"]:
+                            shutil.copy(path.join(vs_dll_dir, vs_dll), servo_exe_dir)
+                    msvc_deps = [
+                        "vcruntime140.dll",
+                        "msvcp140.dll",
+                        "api-ms-win-crt-runtime-l1-1-0.dll",
+                    ]
+                    for msvc_dll in msvc_deps:
+                        dll = subprocess.Popen(["where", msvc_dll], stdout=subprocess.PIPE)
```

Comment:
Can this use `distutils.spawn.find_executable` instead? If not, please add a comment about why that doesn't work.